data/human/hails.txt

```diff
@@ -2195,6 +2195,364 @@ phrase "friendly civilian"
+		"mining lasers. They yield you more minerals per asteroid"
+		"mining lasers. Your mineral yield will go up noticibly with a few of those"
+		"mining lasers. They're more precise than other lasers and preserve more minerals when the asteroid breaks up"
+		"mining lasers so you can finish your mining run before the pirates get to you"
+		"mining lasers" 5
+		"heavy lasers. Their high range makes it easier to destroy asteroids, and you can also use them to scare off pirates"
+		"heavy lasers. Because they have a greater damage output, they're faster at destroying asteroids"
+		"heavy lasers" 2
+		"tractor beams so pirates and rival miners can't steal your hard-earned ore"
+		"tractor beams, which make ore collection just that little bit easier"
+		"tractor beams so picking up ore takes less effort"
+		"tractor beams" 3
+
+phrase "miner advice: mineral"
+	word
+		"you've got to "
+		"you need to look around and "
+		"you have to "
+		""
+	word
+		"mine "
+		"target "
+		"go for "
+		"find systems with "
+		"look for "
+	word
+		"the most valuable minerals"
+		"the most valuable asteroids"
+		"rare asteroids"
+		"uranium"
+		"platinum"
+		"gold"
+		"silver"
+		"tungsten"
+		"neodymium"
+		"uranium asteroids"
+		"platinum asteroids"
+		"gold asteroids"
+		"silver asteroids"
+		"tungsten asteroids"
+		"neodymium asteroids"
+		"minerals like uranium"
+		"minerals like platinum"
+		"minerals like gold"
+		"minerals like silver"
+		"minerals like tungsten"
+		"minerals like neodymium"
+
+phrase "miner advice: mineral"
+	word
+		"you've got to "
+		"you should "
+		"you should learn to "
+		"you're better off to "
+		"it's best to "
+		""
+	word
+		"avoid mining "
+		"avoid picking up "
+		"avoid harvesting "
+		"avoid "
+		"dump "
+		"jettison "
+	word
+		"silicon"
+		"iron"
+		"lead"
+		"aluminum"
+		"silicon ore"
+		"iron ore"
+		"lead ore"
+		"aluminum ore"
+		"minerals like silicon"
+		"minerals like iron"
+		"minerals like lead"
+		"minerals like aluminum"
+		"ore like silicon"
+		"ore like iron"
+		"ore like lead"
+		"ore like aluminum"
+	word
+		" and mine other asteroids"
+		" and find more valuable minerals"
+		" so you have space for more profitable minerals"
+		" because they don't sell for very much"
+
+phrase "miner advice: ship"
+	word
+		"you'll want to"
+		"it's a good idea to"
+		"it's advisable to"
+		"the best thing you can do is"
+	word
+		" "
+	word
+		"expand your fleet with"
+		"purchase"
+		"take out a mortgage for"
+		"buy"
+		"go to the shipyard and get"
+		"get"
+	word
+		" a "
+	word
+		"specialized mining ship like a Sunder"
+		"ship fit for mining, like a Sunder"
+		"Syndicate Sunder"
+		"Sunder and two mining drones"
+		"Sunder mining vessel"
+		"good mining ship, like a Sunder"
+		"defensible ship like a Headhunter"
+		"multi-purpose ship like a Headhunter"
+		"defensible ship like an Argosy"
+		"versatile ship like an Argosy"
+		"carrier like an Aerie. You can use the fighters to mine for you"
```

Comment:
Was this behavior actually merged yet? I can't recall, but I thought at one point it was still bound up in the other fighter behaviors PR?

Reply:
It should work, the Sunder and its Mining Drones also depend on it and they were implemented in 0.10.5.